chapters/introduction/introduction.mdx

```diff
@@ -13,23 +13,23 @@ in both live and asynchronous ways, with audio intelligence tools to extract, an
 
 <CardGroup cols={2}>
   <Card
-    title="Configure your account"
+    title="Setup your account"
     icon="gear"
-    href="/chapters/get-started/pages/configure-account"
+    href="/chapters/introduction/getting-started#setup-your-account"
   >
     Setup your account quickly and get your API key.
   </Card>
   <Card
     title="Transcribe an audio file"
     icon="microphone-lines"
-    href="/chapters/get-started/pages/transcribe-audio"
+    href="/chapters/introduction/getting-started#gladias-playground"
```

Comment:
```suggestion
    href="/chapters/introduction/getting-started#gladia-playground"
```
Should we remove the `s` here and in matching anchors?

Reply:
Mintlify is smart enough to handle this